spec/acceptance/zabbix_application_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper_acceptance'
 require 'serverspec_type_zabbixapi'
 
-describe 'zabbix_application type' do
+describe 'zabbix_application type', unless: default[:platform] =~ %r{debian-10-amd64} do
```

Comment:
Rather than skipping, should we just not set the `zabbix_version` and use the default in this test suite (repeated in multiple files)?

Reply:
sadly that won't work. the custom types use the zabbixapi rubygem and that currently only supports older zabbix versions

Reply:
https://github.com/voxpupuli/puppet-zabbix/pull/623 suggests this can work if we update it.

Reply:
I would like to get this merged and work on #623 afterwards